packages/extension-driver-snowflake/src/lib/snowflakeDataSource.ts

```diff
@@ -100,6 +97,78 @@ export class SnowflakeDataSource extends DataSource<any, SnowflakeOptions> {
     }
   }
 
+  public override async export(exportOptions: ExportOptions): Promise<void> {
+    let { directory } = exportOptions;
+    const { profileName, sql } = exportOptions;
+
+    const { connection, pool } = await this.getConnection(profileName);
+    directory = /^\./.exec(directory) ? directory : `./${directory}`;
+
+    // throw error if filepath dir not exist
+    if (!fs.existsSync(directory)) {
+      throw new InternalError(`Directory ${directory} not exist`);
+    }
+
+    try {
+      const stage = '@~/vulcan_cache_stage';
+      const stageFilePath = `${stage}/`;
+      const builtSQL = this.getCopyToStageSQL(sql, stageFilePath);
+      // copy data to a named stage
+      const copyStatement = await this.getStatementFromSQL(
+        connection,
+        builtSQL,
+        []
+      );
+      const statementId = copyStatement.getStatementId();
+      const pattern = `.*${statementId}.*`; // UUID generated by snowflake
+      await this.waitStatement(copyStatement);
+      this.logger.debug('Copied data to internal stage: @~/vulcan_cache_stage');
+
+      // get parquet file from a named stage
+      const absoluteDir = path.resolve(directory);
+      const getStatement = await this.getStatementFromSQL(
+        connection,
+        `GET ${stage} file://${absoluteDir} pattern='${pattern}'`,
+        []
+      );
+      await this.waitStatement(getStatement);
+      this.logger.debug(`Exported parquet files to ${directory}`);
+
+      // remove parquet file from stage
```

Comment:
Suggest rich the comment to describe removing the stage parquet files because we keep them in the stage temporarily for downloading, so it need to remove 😄